src/components/addProductsForm/AddProducts.jsx

```diff
@@ -1,5 +1,181 @@
-import React from "react";
+import React, { useState } from "react";
+import { Form, Button, Row, Col, Container } from "react-bootstrap";
+import "./style.scss";
+import db from "../config/firebaseConfig";
 
 export default function AddProducts() {
-  return <div>add Products</div>;
+  const [hasSizes, setHasSizes] = useState(false);
```

Comment:
hmm, why do all of these start with get? then you get things like "setGet" which is really confusing. i'm assuming you probably want something like "formProductBrand" or evne just "productBrand" is fine here prob